sympy/utilities/lambdify.py

```diff
@@ -297,7 +297,18 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     """
     from sympy.core.symbol import Symbol
     from sympy.utilities.iterables import flatten
-
+    invalid_str_expr = 0
+    invalid_str_args = 0
+    if isinstance(expr, str):
+        if not tuple(expr.replace(',','')) == tuple(expr.split(',')):
+            invalid_str_expr = 1
+    if isinstance(args, str):
+        if not tuple(args.replace(',','')) == tuple(args.split(',')):
+            invalid_str_args = 1
+    if  invalid_str_expr == 1 :
```

Comment:
Encoding logical state as integers is what you do if you do not have a ```boolean``` data type.
In Python, use ```True``` and ```False``` instead of ```0``` and ```1```.
This allows you to write ```if invalid_str_expr:``` instead of ```if invalid_str_expr == whatever:```, which is more concise and expresses the intent more directly.